### PR TITLE
AssetMeta - Timeseries tab

### LINKED ISSUE #269: Timeseries Tab in AssetMeta
Add new tab `Timeseries` in `AssetMeta` component.
The tab content should show a list of timeseries and if a user clicks on one `TimeserieChart` will be shown (see OptInt for reference).

### PR DESCRIPTION
**Pull request by xitman78**
- Introduced component `TimeseriesChartMeta`
- Added tab "Timeseries" in `AssetMeta` component
- Unit Tests
- Documentation

Closes #269

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-278.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-278.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-278.gearboxjs.preview.cogniteapp.com

File: src/components/AssetMeta/components/TimeseriesPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import React from 'react';
+import styled from 'styled-components';
+import {
+  TimeseriesChartMeta,
+  TimeseriesChartMetaProps,
+} from '../../TimeseriesChartMeta';
+
+export interface MetaTimeseriesProps
+  extends Omit<TimeseriesChartMetaProps, 'timeseries'> {
+  noTimeseriesSign?: string;
```
Comment (follan) [inline src/components/AssetMeta/components/TimeseriesPanel.tsx]: Can you use strings pattern we use in other components for custom strings?

**Pull request merged**